pipeline/talend_test/test_configs/anfog_rt.yaml

```diff
@@ -0,0 +1,22 @@
+
+
+name: anfog_rt
+type: pipeline_version_2
+po: bene
+actions:
+  - type: ADD
+    files:
+    - dest: /mnt/ebs/incoming/ANFOG/processed
+      local_file: BENE TO ADVISE
+create_schema: true
+talend_log_file: /mnt/ebs/log/pipeline/process/tasks.ANFOG_RT.log
+talend_jobs:
+  - anfog_rt-anfog_rt
+database_schemas:
+  -
+    name: anfog_rt-anfog_rt
+    tables:
+      - name: anfog_rt_trajectory_map
+        exclude_columns: []
+      - name: anfog_rt_trajectory_map
```

Comment:
Did you mean `anfog_rt_trajectory_data` here?

Reply:
I sure did :) sorry, copy/paste error by me.